### DISCUSSION TITLE
Specifying the NubesGen version on the NubesGen website

### DISCUSSION THREAD

**Discussion by agoncal** [microsoft/NubesGen#136] [Ideas]
NubesGen is now in 0.1 and will soon be in 0.1.1 (or 0.2). It would be nice if in the [NubesGen website](https://nubesgen.com) this version was specified somewhere. This way, we would know if the templates had changed since a previous download. WDYT ?

**Comment by cmaneu** [comment]
As we have a little Hackathon this week, I've created the issue #180 for tracking this idea 😉
